packages/react/src/components/Tabs/TabPanel.tsx

```diff
@@ -1,19 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useId } from 'react-id-generator';
 
 interface TabPanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  panelref: React.RefObject<HTMLDivElement>;
```

Comment:
Why not just `ref`? You would need to use `React.forwardRef` here. We've tried to move away from having named refs, except in cases where we wanted to maintain backwards compatibility (see [Button](https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Button/index.tsx)).

```suggestion
  ref: React.RefObject<HTMLDivElement>;
```